src/style.css

```diff
@@ -21,12 +21,28 @@ header {
 
 ul {
   list-style: none;
+  display: flex;
+  gap: 5vw;
 }
 
 a {
   text-decoration: none;
 }
 
+li {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  width: 8vw;
+  height: 5vh;
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  border: 1px solid transparent;
+}
+
+li:hover {
+  border: 1px solid grey;
+}
+
```

Comment:
- [ ] It looks like your changes to the CSS file affected the popUP

- Please Try styling the menu links using the class name. Adding CSS rules directly to the HTML tags affects other parts of the code with the same tag. 
- 
![pop-up-foodHub](https://user-images.githubusercontent.com/107632780/203726202-6d4c8660-9571-47d2-91cd-29a738664398.png)